doc/source/dev/conduct/report_handling_manual.rst

```diff
@@ -70,6 +84,49 @@ action deemed necessary.
 How the committee will respond to reports
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+There are two tracks of response; one is specific to clear and severe breaches
+(as defined below).
+
+Clear and severe breach
++++++++++++++++++++++++
+
+We know that it is painfully common for internet communication to start at or
+devolve into obvious and flagrant abuse.  We define a clear and severe breach
+as being: personal threats, violent, sexist or racist language.
+
+In what follows, we define a Scipy contributor as someone who has done any of
+the following, at least a week prior to the incident in question:
+
+* sent more than one substantial email to the mailing list;
+* submitted a substantial pull request;
+* submitted a substantial issue report.
+
+If any member sees a communication that qualifies as a personal threat, or
+violent, sexist or racist language, we ask them to contact a member of the
```

Comment:
This is a bit odd, we're already in the "responding to reports" section here.

Reply:
Right - maybe it should go elsewhere.

Reply:
Moving this to the reporting guidelines section of the main CoC doc.